### PR TITLE
Jenkins Project Setup Update

### PR DESCRIPTION
**Pull request by ButkiewiczP**
This is a small update to the Jenkins instructions for both iOS and Android projects. The new instructions simply include notes about creating projects in the appropriate sub-folders.

Eventually, creating projects in folders will be limited by your security level, but for now, it's easy enough to just tell people where to create their projects.

### CODE REVIEW

**Comment by gsintrepid** [comment]
@ButkiewiczP can you also update the [readme](https://github.com/IntrepidPursuits/sherpa/blob/master/readme-template.md) with build/cobertura status?

**Comment by ButkiewiczP** [comment]
@gsintrepid Readme template is now updated.

File: readme-template.md
Comment (gsintrepid) [inline readme-template.md]: @ButkiewiczP Thanks!
Just to confirm, would `http://ci.intrepid.io:9913/jenkins/cobertura/ios-template/` work, or should it be `https://ci.intrepid.io:9913/jenkins/cobertura/iOS/job/ios-legacy/job/ios-template`?

Comment (ButkiewiczP) [reply readme-template.md]: Good catch. Updated in the latest commit

File: ios/ios_project_setup.md
Comment (djbrooks111) [inline ios/ios_project_setup.md:7]: Is this actually necessary? What about Keolis where its project is one level below root?

Comment (ButkiewiczP) [reply ios/ios_project_setup.md:7]: Keolis was inherited with a poor project structure that has caused us some pains in the past. This doc is supposed to explain how to set up a new Xcode project and repository, so I feel like the instructions should point to what is easiest for us.

File: ios/ios_jenkins_pipeline_guide.md
Comment (brightredchilli) [inline ios/ios_jenkins_pipeline_guide.md:42]: maybe worth clarifying, since project names change - is this the project name? or workspace name?

**Review by brightredchilli — APPROVED** [review]

File: ios/ios_jenkins_pipeline_guide.md
Comment (ButkiewiczP) [reply ios/ios_jenkins_pipeline_guide.md:42]: I didn't want to write `workspace` because every xcode project may not have a workspace. It IS the name of your `.xcodeproj` or `.xcworkspace` -- but what's the best way to say that here. 

And shouldn't the project name never change?

**Comment by ButkiewiczP** [comment]
Any other comments on this?

**Comment by ButkiewiczP** [comment]
I don't want to keep badgering people about this. 

But I most definitely will.

With the apprentices starting, and new projects on the horizon, this really needs to be merged. Waiting on one more approval, or any other comments

File: ios/ios_jenkins_pipeline_guide.md
Comment (petey3) [inline ios/ios_jenkins_pipeline_guide.md]: "By default, all iOS projects on Jenkins will load the Intrepid Pipeline Library"

How does this get loaded? If I just added a repo on the github org how do I get the default Jenkinsfile @ButkiewiczP

Comment (ButkiewiczP) [reply ios/ios_jenkins_pipeline_guide.md]: There may be some confusion between the Library and the Jenkinsfile. 

A Jenkinsfile is just a text file that's created on a per-project basis, and added to the repo by someone on the team. The Jenkinsfile usually has some Java/Groovy code which defines the pipeline (the build code) to be run for the project. There's a bunch of default functions (like `sh` which runs a shell command) available for all Jenkinsfiles.

Our library extends the available methods to use in the Jenkinsfile. In particular, it provides some classes which define the Project model (`XcodeProject`), and a class which acts as a wrapper around all of the Java/Groovy pipeline code you _would_ need to write, if we didn't have a library to wrap it with.

This allows us to greatly simplify the Jenkinsfile that team leads need to write, by only having to define a single model, and then passing it to a function (the wrapper function `xcodePipeline`)

Comment (ButkiewiczP) [reply ios/ios_jenkins_pipeline_guide.md]: I'm thinking it might be a good idea to link to a few more example `Jenkinsfiles`

Comment (petey3) [reply ios/ios_jenkins_pipeline_guide.md]: Ah I see - I think my confusion was with the following:
```
This gives you immediate access to all the packages and classes within the library if you need to write a non-standard Jenkinsfile. Create the file by adding a text file named `Jenkinsfile` to your repo.
```
Specifically, it read that I would only need to create a Jenkinsfile if I wanted non-standard functionality. Otherwise, I would have _waves hands_  **poof** my Jenkinsfile out the gate

**Comment by ButkiewiczP** [comment]
I'm merging this today if there are no more comments by Noon O'Clock

**Pull request merged**
